src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/PhysicalAddress.cs

```diff
@@ -103,22 +103,39 @@ public byte[] GetAddressBytes()
             return (byte[])_address.Clone();
         }
 
-        public static PhysicalAddress Parse(string address)
+        public static PhysicalAddress Parse(string address) => address != null ? Parse(address.AsSpan()) : None;
```

Comment:
It seems before the PR Parse(null) throw.

For string.Empty we throw too so we could use implicit conversion string->ReadOnlySpan<char>
```c#
public static PhysicalAddress Parse(string address) => Parse(address);
```

Reply:
> It seems before the PR Parse(null) throw.

No, it returned null:
https://github.com/dotnet/runtime/pull/1057/files#diff-74838e07dff5967e2e1173264255be64L112-L115

Reply:
I get the exception in PowerShell console (.Net Core 3.1)
![image](https://user-images.githubusercontent.com/22290914/71917460-f5b50b80-31a1-11ea-8460-59e6b15a0127.png)

Reply:
I don't know what PowerShell is passing there, but passing in `null` returns None:
```
using System;
using System.Net.NetworkInformation;

class Program
{
    static void Main()
    {
        Console.WriteLine(PhysicalAddress.Parse(null) == PhysicalAddress.None);
    }
}
```
outputs:
```
True
```
That's the case for .NET Framework as well.

Reply:
@stephentoub Thanks! I will open new issue in PowerShell repo.